retail/retail-java-applications/dashboards/retail_pipeline.json

```diff
@@ -0,0 +1,271 @@
+{
```

Comment:
dashboards will probably need a sub folder which is cloud-monitoring in case in the future there will be other examples.